OpenRA.Server/Program.cs

```diff
@@ -90,21 +90,28 @@ static void Run(string[] args)
 				var endpoints = new List<IPEndPoint> { new IPEndPoint(IPAddress.IPv6Any, settings.ListenPort), new IPEndPoint(IPAddress.Any, settings.ListenPort) };
 				var server = new Server(endpoints, settings, modData, ServerType.Dedicated);
 
+				Widgets.Ui.InitializeTranslation(); // for server messages
```

Comment:
```suggestion
				// For server messages
				Widgets.Ui.InitializeTranslation();
```

Reply:
You do realize that the comment reads as a sentence together with the code? Anyway, this may not even be required after https://github.com/OpenRA/OpenRA/pull/19975.